### ISSUE TITLE
about the flops in paper

### ISSUE THREAD

**Issue by suzhenghang** [arunos728/MotionSqueeze#4]
Hi, could you share the code about computing the flops in your paper? Thanks in advance.

**Comment by KimManjin** [comment]
Hi,
We adopted Pytorch code, which is available online ([link](https://github.com/Lyken17/pytorch-OpCounter)), to count FLOPs.
We simply added a few lines in to count FLOPs for computing correlation as follows:
```
def count_correlation(m, x, y):
    b,c,h1,w1 = x[0].size()
    ops_per_instance = 2*c-1
    total_ops = y.numel()*ops_per_instance
    m.total_ops += torch.Tensor([int(total_ops)])
```
Note that we neglect to count FLOPs for few operations, eg. L2-normalization and masking gaussian kernels, of which computation cost is relatively small.
We committed our FLOPs counter code to our repository.

Thanks for you attention to our work and questions are always welcomed.

Thanks.

**Comment by AliceCrystal** [comment]
Thanks for your great job! Could you share the configuration of TSM ResNet-18, including training eopchs, batch_size, initial learning rate and learning rate decay ?   For the Something-Something V1&V2.

**Comment by arunos728** [comment]
We updated the FLOPs code.
